SpreedlySdk.xcodeproj/project.pbxproj

```diff
@@ -801,7 +800,6 @@
 				F076C6D90DB36BC555C246AC /* BankAccountInfo.swift in Sources */,
 				A60A0240278796000044BD69 /* SpreedlyVersion.swift in Sources */,
 				F076CD7FF9E8D8C63D808B1A /* CreditCardInfo.swift in Sources */,
-				442A2D2E48133A09DBBD9299 /* Package.swift.template in Sources */,
```

Comment:
Removed this non-Swift file from the list of Compile Sources to fix the `no rule to process file...` warning for this file.

Src: https://stackoverflow.com/questions/22778021/xcode-warning-no-rule-to-process-file-when-build-phases-has-this-file